### PR TITLE
xray merger interface 

### PR DESCRIPTION
**Pull request by krzesi-mir**

### CODE REVIEW

File: xray/src/bin/merge_xray_quadtrees.rs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+/// of each octree belongs to the same level of the final
+/// octree. 
+struct CommandlineArguments {
+    /// Directory with partial xray quadtrees.
+    #[structopt(parse(from_os_str))]
+    directory: PathBuf,
+    /*
+    /// Paths to each quadtree's metadata protobuf file.
+    metadata_proto_filenames: Vector<Pathbuf>
+    */
```
Comment (krzesi-mir) [inline xray/src/bin/merge_xray_quadtrees.rs]: What do you think: should this binary scan the directory and look for metadata protos on its own, or should the user pass  as an argument the paths to those protos?

Comment (feuerste) [reply xray/src/bin/merge_xray_quadtrees.rs]: I think passing the directories is enough, with something like https://docs.rs/globwalk/0.8.0/globwalk/ we can easily identify the *.pb files. Going a step further, once we have dozens or hundreds of partial quadtrees, we may instead pass one single file containing all the paths...

File: xray/src/bin/merge_xray_quadtrees.rs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+use std::path::{PathBuf};
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "merge_xray_quadtrees")]
+/// Merge partial xray octrees. We assume that the root
+/// of each octree belongs to the same level of the final
+/// octree.
```
Comment (feuerste) [inline xray/src/bin/merge_xray_quadtrees.rs]: octree(s) -> quadtree(s)

Comment (krzesi-mir) [reply xray/src/bin/merge_xray_quadtrees.rs]: oups, thanks.

**Comment by krzesi-mir** [comment]
@googlebot I fixed it.

**Pull request closed without merging**